scripts/nightly_turing.sh

```diff
@@ -40,64 +41,88 @@ HOSTNAME="turing"
-git pull origin master
+git checkout main
+git pull origin main
 make clean
 eval $(opam env)
 
+#readable divider
+echo "-------------------------------------------------------START-----------------------------------------------------------------------"
+echo $(date)
+echo "-------------------------------------------------------START-----------------------------------------------------------------------"
+
 #sequential benchmarks
 TAG='"macro_bench"' make run_config_filtered.json
-RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/4.12.0+stock.bench
-RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/4.12.0+domains+effects.bench
+USE_SYS_DUNE_HACK=1 RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/5.00.0+trunk.bench
+USE_SYS_DUNE_HACK=1 RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/5.00.0+stable.bench
+#RUN_CONFIG_JSON=run_config_filtered.json make ocaml-versions/5.00.0+domains.bench
 
-cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/4.12.0+domains+effects_1.orun.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_412_DOMAINS_EFFECTS
-cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/4.12.0+stock_1.orun.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_412_STOCK
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+trunk_1.orun.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_TRUNK
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+stable_1.orun.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_STABLE
+#cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+domains_1.orun.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/sequential/$HOSTNAME/$TIMESTAMP/$OCAML_MULTICORE
 rm -rf _results/
 
 #parallel benchmarks
 TAG='"macro_bench"' make multicore_parallel_run_config_filtered.json
+	USE_SYS_DUNE_HACK=1 \
+	RUN_BENCH_TARGET=run_orunchrt \
+	BUILD_BENCH_TARGET=multibench_parallel \
+	RUN_CONFIG_JSON=multicore_parallel_run_config_filtered.json \
+	make ocaml-versions/5.00.0+trunk.bench
 
-RUN_BENCH_TARGET=run_orunchrt \
+	USE_SYS_DUNE_HACK=1 \
+	RUN_BENCH_TARGET=run_orunchrt \
 	BUILD_BENCH_TARGET=multibench_parallel \
 	RUN_CONFIG_JSON=multicore_parallel_run_config_filtered.json \
-	make ocaml-versions/4.12.0+domains+effects.bench
+	make ocaml-versions/5.00.0+stable.bench
 
-cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/4.12.0+domains+effects_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_412_DOMAINS_EFFECTS/
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+trunk_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_TRUNK/
+cp -a $SANDMARK_NIGHTLY_DIR/sandmark/_results/5.00.0+stable_1.orunchrt.summary.bench $SANDMARK_NIGHTLY_DIR/sandmark-nightly/parallel/$HOSTNAME/$TIMESTAMP/$OCAML_STABLE/
 rm -rf _results/
 
```

Comment:
Now that https://github.com/ocaml-bench/sandmark/pull/283 has been merged. Please add the folllowing to both the scripts:
```
SANDMARK_NIGHTLY_DIR=$SANDMARK_NIGHTLY_DIR bash run_all_custom.sh
```